### PR TITLE
Search Listing

### PR DESCRIPTION
**Pull request by jeffdowdle**
…work alongside app search

**Issue**:

### What I did

- 
- 

### How to test

- 
- 

#### For all PR's

- [ ] I've added relevant changes to the project Readme if needed.
- [ ] I've updated the documentation site as needed.
- [ ] I have added unit tests to cover my changes (if not applicable, please state why in a comment)

#### For new components only

- [ ] I have added a story covering all variants
- [ ] I have checked a11y tab in storybook passes
- [ ] Any events are emitted on the event bus

### CODE REVIEW

**Review by waitingallday — APPROVED** [review]

**Comment by dylankelly** [comment]
@jeffdowdle in https://github.com/dpc-sdp/ripple-framework/blob/feature/search-listing/packages/ripple-tide-search/components/TideSearchPage.vue#L205 I think we should add another error message for when there is no search term. 

With site search this makes sense, but with scoped search pages there could easily be no results with no search term. But I'm happy to raise this in another ticket so we can get some copy written.

File: packages/ripple-tide-search/components/TideSearchListingPage.vue [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+              <p class="rpl-type-h3">
+                Sorry! Something went wrong. Please try again later.
+              </p>
+            </RplContent>
+          </div>
+          <RplContent
+            v-else-if="!isBusy && !results?.length"
+            data-component-type="search-listing-no-results"
+          >
+            <p class="rpl-type-h3">
```
Comment (dylankelly) [inline packages/ripple-tide-search/components/TideSearchListingPage.vue:291]: can we add a check for whether there is a query in `route.query.q` and give a different message here? It wouldn't make sense to get this message if there is no search term.

I'll add a bug ticket for it so we can get some copy for it.

File: packages/ripple-tide-search/components/TideSearchFilters.vue [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  <RplForm
+    id="tide-search-filter-form"
+    class="rpl-u-margin-t-6"
+    @submit="handleFilterSubmit"
+  >
+    <div class="rpl-grid rpl-grid--no-row-gap tide-search-filters">
+      <div
+        v-for="filter in filterInputs"
+        :key="filter.id"
+        class="rpl-col-12 rpl-col-6-m"
```
Comment (dylankelly) [inline packages/ripple-tide-search/components/TideSearchFilters.vue:11]: We will need to change this to make the classes come from the filter config. Id like to change this component to use the schema instead, Im not keen on the[ wrapper component](https://github.com/dpc-sdp/ripple-framework/pull/700/files#diff-17908af448800f3a3e3758e7b4b0aa179de596d484ca8a72c100fb6a6a1b5de5) in `packages/ripple-tide-search/components/global/TideSearchFilterDropdown.vue` - But let's add this in a follow up when add more filter form elements

File: examples/nuxt-app/test/features/search-listing/filters.feature [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+      | termsFilter    |
+      | functionFilter |
+
+    Then the search input should have the value ""
+    Then the search listing dropdown field labelled "Raw filter example" should have the value "Select a pet"
+    Then the search listing dropdown field labelled "Term filter example" should have the value "Select a colour"
+    Then the search listing dropdown field labelled "Terms filter example" should have the value "Select a colour"
+    Then the search listing dropdown field labelled "Custom function filter example" should have the value "Select a status"
+
+# @mockserver
```
Comment (dylankelly) [inline examples/nuxt-app/test/features/search-listing/filters.feature]: This is failing because the filters aren't visible. Have updated

**Review by dylankelly — APPROVED** [review]
some follow up tasks, but lets get it in to develop. Great work

**Pull request merged**